File: simpson-katy/breaking_chocolate_problem.js
function breakChocolate(n,m) {
    if (n === 0 || m === 0) {
      return 0
    } else {
      return n * m - 1;  
    }
  }
  
  console.log(breakChocolate(5, 5));
  console.log(breakChocolate(1, 1));
  console.log(breakChocolate(3, 4));

File: thomas-zachary/breaking-chocolate-problem.py
def break_chocolate(n, m):
#   python
#   given a chocalate bar of (n * m)
#   break into 1 * 1 squares
#   if no breaks necessary return 0
    x = n * m
    if x < 2:
        return 0
    else:
        return x - 1